.github/workflows/gh-actions-prover-benches.yml

```diff
@@ -0,0 +1,30 @@
+name: GH Actions Prover Benches on PR
+on:
+  pull_request:
+    types: [ labeled ]
+jobs:
+  Prover_benchmarking_Automation:
+    if: ${{ github.event.label.name == 'exec_prover_bench' }}
```

Comment:
I'd change to `benchmarks` or something simpler

Reply:
@CPerezz sure no issues.